GVRf/Framework/framework/src/main/jni/objects/components/render_data.cpp

```diff
@@ -36,9 +36,14 @@ const RenderPass* RenderData::pass(int pass) const {
 }
 
 void RenderData::set_mesh(Mesh* mesh) {
-    mesh_ = mesh;
-    mesh->add_dirty_flag(dirty_flag_);
-    *dirty_flag_ = true;
+    if (mesh != nullptr) {
+        mesh_ = mesh;
+        mesh->add_dirty_flag(dirty_flag_);
+        *dirty_flag_ = true;
+    }
+    else {
+        LOGE("RenderData::set_mesh mesh cannot be NULL");
```

Comment:
It should probably std::terminate in addition to the log trace. Or just use the FAIL macro from gvr_log.h.